routers/misc.py

```
@@ -0,0 +1,42 @@
+
+@misc_router.post('/summary')
+@x_super_team
+@x_app_key
+@jwt_token
+async def create(request: Request):
+    try:
+        data = await request.form()
+
+        required_fields = ['messages']
+        if not check_required_fields(data, required_fields):
+            raise HTTPException(status_code = 400, detail = f"An error occurred: missing parameter(s)")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)